lib/Pod/Simple/XHTML.pm

```diff
@@ -774,12 +774,12 @@ sub resolve_pod_page_link {
 Resolves a man page link target and numeric section to a URL. The resulting
 link will be returned for the above examples as:
 
-    http://man.he.net/man5/crontab
-    http://man.he.net/man1/crontab
+    https://man7.org/linux/man-pages/man5/crontab.5.html
+    https://man7.org/linux/man-pages/man1/crontab.1.html
 
 Note that the first argument is required. The section number will be parsed
 from it, and if it's missing will default to 1. The second argument is
-currently ignored, as L<man.he.net|http://man.he.net> does not currently
+currently ignored, as L<man7.org|https://man7.org/linux/man-pages> does not currently
```

Comment:
This is inaccurate. man7.org does have linkable sections, e.g. https://man7.org/linux/man-pages/man5/crontab.5.html#EXAMPLE_CRON_FILE.

Reply:
Good to know, it wasn't that obvious since headings are not clickable.
Removed mentioning that, it can be implemented in a separate PR in the future.